.golangci.yml

```diff
@@ -9,6 +9,9 @@ linters:
   enable:
     - gofmt
     - unparam
+    - revive
```

Comment:
why not `typecheck`?

Reply:
actually, I'm currently using a more exhaustive list of linters:
```
linters:
  enable:
    - megacheck
    - govet
    - gocyclo
    - unused
    - gofmt
    - revive
    - misspell
    - exportloopref
  presets: # groups of linters. See https://golangci-lint.run/usage/linters/
    - bugs
    - unused
  disable: 
    - golint # deprecated, use 'revive'
    - scopelint # deprecated, use 'exportloopref'
 ```
and `typecheck` is part of the `bugs` preset. See https://golangci-lint.run/usage/linters/

Reply:
and I think that I can even remove linters already included in the `presets` from the `enabled` list